### PR TITLE
[13.0][MIG] sale_commercial_partner

### PR DESCRIPTION
**Pull request by kongrattapong**
Standard Migration

### CODE REVIEW

**Comment by sa3m** [comment]
Hello,
Can we help this PR to be merged ?

**Review by hparfr — APPROVED** [review]
LGTM

**Comment by hparfr** [comment]
@sa3m Yes, you can review comment and/or approve this PR.

File: sale_commercial_partner/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
```
Comment (newtratip) [inline sale_commercial_partner/models/__init__.py]: No need for copyright in init file.

File: sale_commercial_partner/views/sale.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            </field>
+        </field>
+    </record>
+
+    <record id="view_sales_order_filter" model="ir.ui.view">
+        <field name="name">commercial.partner.sale.order.search</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_sales_order_filter"/>
+        <field name="arch" type="xml">
+            <filter context="{'group_by': 'partner_id'}" position="before">
```
Comment (newtratip) [inline sale_commercial_partner/views/sale.xml]:
```suggestion
            <filter name="customer" position="before">
```

File: sale_commercial_partner/views/sale.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<odoo>
```
Comment (newtratip) [inline sale_commercial_partner/views/sale.xml]: Please rename file to sale_views.xml https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

**Review by newtratip — CHANGES_REQUESTED** [review]
@kongrattapong change a little.

**Comment by kongrattapong** [comment]
> @kongrattapong change a little.

Thank for review, fixed.

**Comment by kongrattapong** [comment]
@newtratip Thank for review, fixed.

**Review by newtratip — APPROVED** [review]
It seem OK :+1:

**Comment by newtratip** [comment]
@kongrattapong Please merge 2 commit to 1 commit by using squash.

**Comment by simahawk** [comment]
@kongrattapong ping on https://github.com/OCA/sale-workflow/pull/1023#issuecomment-600702608 ;)

**Review by sebalix — APPROVED** [review]

**Pull request merged**